cms/static/js/views/settings/main.js

```diff
@@ -63,6 +63,15 @@ var DetailsView = ValidatingView.extend({
         var imageURL = this.model.get('course_image_asset_path');
         this.$el.find('#course-image-url').val(imageURL);
         this.$el.find('#course-image').attr('src', imageURL);
+        if(this.model.get('entrance_exam_enabled') == 'true'){
```

Comment:
Would you mind adding spaces around the parentheses like we do everywhere else. I don't know if this will fail our JSHint file or not (probably not), but it would be preferable to be be consistent within this code.